### PR TITLE
Document shouldExecute script config option

### PR DESCRIPTION
**Pull request by DoodleBobBuffPants**

### CODE REVIEW

File: documentation/configuration/scriptconfigfiles.md [hunk 1]
```diff
@@ -36,6 +36,13 @@ encoding=ISO_8859_1
 # Manually determine whether or not to execute this migration in a transaction. This is useful for
 # databases like PostgreSQL and SQL Server where certain statements can only execute outside a transaction.
 executeInTransaction=false
+
+# Whether this migration should be executed or ignored. Valid values are 'true', 'false', or 'A=B' where
```
Comment (juliahayward) [inline documentation/configuration/scriptconfigfiles.md]: just needs the A==B change now...

File: documentation/configuration/scriptconfigfiles.md [hunk 2]
```diff
@@ -36,6 +36,13 @@ encoding=ISO_8859_1
...
+# A and B are boolean values and not themselves expressions. This migration is executed if the boolean
+# expression evaluates to true, ignored if it evaluates to false, and throws an exception if the expression
+# is invalid. Placeholder replacement is also supported in the expression.
+# Flyway Teams only
+shouldExecute=${should_execute}=true
```
Comment (juliahayward) [inline documentation/configuration/scriptconfigfiles.md]: ... and here

**Review by juliahayward — APPROVED** [review]

**Comment by MikielAgutu** [comment]
I think this certainly needs a more in-depth explanation, and examples of usage (blog post?)

**Comment by DoodleBobBuffPants** [comment]
> I think this certainly needs a more in-depth explanation, and examples of usage (blog post?)

I don't mind making this the topic of my blog post, whenever that is due 😬

**Comment by MikielAgutu** [comment]
> don't mind making this the topic of my blog post, whenever that is due 😬

When the release goes out? 😉

**Comment by DoodleBobBuffPants** [comment]
> When the release goes out? 😉

Let's not get ahead of schedule, I'll make it my new year's gift instead 😉

**Pull request merged**